### PR TITLE
Adjusted popularity measure (bug 930902, 1116564)

### PR DESCRIPTION
**Pull request by robhudson**
This changes how we calculate popularity. It stop using the `Installed` table, which only records installs from logged in users, and moves to using Monolith which tracks installs from all users. It also does a rolling 90 day popularity so newer apps that become popular can also come into play.

I'm removing `total_downloads` and `weekly_downloads` here as well as they weren't being used any longer.

### CODE REVIEW

File: migrations/886-rm-weekly-downloads.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE addons DROP COLUMN weeklydownloads;
```
Comment (washort) [inline migrations/886-rm-weekly-downloads.sql]: +1 less stuff in `addons`

File: migrations/887-rm-total-downloads.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+ALTER TABLE addons DROP COLUMN totaldownloads;
```
Comment (washort) [inline migrations/887-rm-total-downloads.sql]: i guess that's +2 then

File: mkt/lookup/views.py [hunk 3]
```diff
@@ -447,9 +447,10 @@ def _app_downloads(app):
              'alltime': 0}
 
     # Webapps populate these fields via Monolith.
+    # TODO: Update this to use monolith or link to stats or use 90d numbers?
     stats['last_24_hours'] = 'N/A'
-    stats['last_7_days'] = app.weekly_downloads
-    stats['alltime'] = app.total_downloads
+    stats['last_7_days'] = 'N/A'
```
Comment (eviljeff) [inline mkt/lookup/views.py]: Um... TODO to implement something this patch removed?  What's the blocker?

Comment (robhudson) [inline mkt/lookup/views.py]: @eviljeff I guess just knowing what would be good info there? Also, I removed the totals from the reviewer tools. If you'd like to have the 90-day popularity count I can replace it with that.

Comment (washort) [inline mkt/lookup/views.py]: this is the only part that confuses me.

**Comment by washort** [comment]
r+

File: mkt/lookup/templates/lookup/app_summary.html [hunk 4]
```diff
@@ -108,12 +108,6 @@
             {% endfor %}
           </ul>
         </dd>
-        <dt>{{ _('Downloads') }}</dt>
-        <dd class="pretty-list">
-          {% for key, heading in summaries %}
-            <div>{{ heading }}: {{ downloads[key] }}</div>
-          {% endfor %}
-        </dd>
```
Comment (robhudson) [inline mkt/lookup/templates/lookup/app_summary.html]: @eviljeff Updated to just remove this section in the lookup tool. We have a link to statistics on this page already and that will be more accurate than this ever was.

**Comment by robhudson** [comment]
Merged: https://github.com/mozilla/zamboni/commit/7388ab8d90839d6c3483eb15a783e05903e0b707

**Pull request closed without merging**